tools/ci_build/github/linux/docker/scripts/training/azure_scale_set_vm_mount_test_data.sh

```diff
@@ -0,0 +1,30 @@
+#! /usr/bin/env bash
+if [ -d "/bert_data" ]; then
+sudo umount /bert_data
+fi
+if [ -d "/bert_data" ]; then
+sudo rm -rf /bert_data
+fi
+
+sudo mkdir /bert_data
+
+if [ ! -d "/etc/smbcredentials" ]; then
+sudo mkdir /etc/smbcredentials
+fi
+
+if [ -f "/etc/smbcredentials/orttrainingtestdata.cred" ]; then
+sudo rm /etc/smbcredentials/orttrainingtestdata.cred
+fi
+
+# to create orttrainingtestdata.cred, I have to do: 'sudo bash -c ...'
+sudo bash -c 'echo "username=orttrainingtestdata" >> /etc/smbcredentials/orttrainingtestdata.cred'
```

Comment:
fyi, another way to write to file that needs root permissions:
https://unix.stackexchange.com/a/4337

Reply:
would suggest doing that to avoid having to update the file permissions